src/app/clusters/service-area-server/service-area-cluster-objects.h

```diff
@@ -0,0 +1,323 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <lib/support/CommonIterator.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace ServiceArea {
+
+inline constexpr size_t kLocationtNameMaxSize   = 64u;
```

Comment:
This should be 128. See file Data-Model.adoc in https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/8933/files

Reply:
This is correctly defined in home-location.xml - still can't find anyway to make use of that. :wine_glass: